### DISCUSSION TITLE
Connect to multiple servers

### DISCUSSION THREAD

**Discussion by pavlo-yashchuk** [FreeOpcUa/python-opcua#1411] [Q&A] [answered]
Hello!

How one client can connect to multiple servers? 
For example, I have 2 or more servers( 10.14.1.10 and 10.14.2.10) with the same node and objects
Can one client to connect to these servers? Maybe 2 different connections in parallel
but how exactly it can be?

**Comment by AndreasHeine** [comment]
you need to have seperate client instances!

**Comment by swamper123** [comment, marked as answer]
Make two client instances, each one connecting to one address and this should work (if this was your intention).

**Comment by pavlo-yashchuk** [comment]
@AndreasHeine @swamper123 
Thanks!! It is what i need ^^
